### DISCUSSION TITLE
Unable to change Dolphin's Close Tab Button size

### DISCUSSION THREAD

**Discussion by FGD-Garuda** [tsujan/Kvantum#965] [Q&A]
Hi there!

I find the close tab button a bit small in Dolphin, I always need to aim for it.

![image](https://github.com/tsujan/Kvantum/assets/35015576/fcecf6ca-911f-4105-9592-130da51f15f9)

I found those buttons are located in `/usr/share/Kvantum/Sweet/Sweet.svg` as I'm using the Sweet Theme.
The element names are `tab-close-*`

I am unsure the size is set by Sweet Theme or something in Kvantum itself.

No matter the size of the elements I set them to in the .SVG, the buttons never change. I can change color, but not the size.

Tnx!

**Comment by tsujan** [comment]
Does the size change in other apps?

**Comment by FGD-Garuda** [comment]
Good question. I only have Dolphin using those specific buttons, maybe cuz they are for tabs inside the app and not the standard Plasma close button located in the Window Title bar.

**Reply by tsujan** [reply]
Try `kvantumpreview` →  last tab. It's there for such tests ;)

**Reply by FGD-Garuda** [reply]
Oh wow, that thing is cool to preview everything! Will definitely use it from now on! :D

Those tab-close-* buttons are still small in there too.

![image](https://github.com/tsujan/Kvantum/assets/35015576/c1328b06-f07f-4e8c-9bc9-b6a46df2afdf)

**Reply by tsujan** [reply]
Yes, they're small. Change `indicator.size` under the `[Tab]` section of the  kvconfig file of the theme you use and run `kvantumpreview` again. It should work.

**Reply by FGD-Garuda** [reply]
Ha!
This particular theme has a Tab section that inherits from another section, so I changed it in that `PanelButtonCommand` section and it works!

I'll copy the kvconfig to my own, so a Sweet update will not override my change.

Thanks so much, @tsujan !! 😄

**Reply by tsujan** [reply]
> This particular theme has a Tab section that inherits from another section

You can still add `indicator.size=...` to it.

**Reply by tsujan** [reply]
I mean, without touching `PanelButtonCommand`.

**Comment by FGD-Garuda** [comment]
Closing
